web/app/models/build.ts

```diff
@@ -37,23 +44,25 @@ export class Build {
   readonly sha: string;
   readonly shortSha: string;
   readonly date: Date;
+  readonly artifacts: Artifact[];
 
-  constructor(buildSummary: BuildResponse) {
-    this.number = buildSummary.number;
-    this.projectId = buildSummary.project_id;
-    this.description = buildSummary.description;
-    this.trigger = buildSummary.trigger;
-    this.lane = buildSummary.lane;
-    this.platform = buildSummary.platform;
-    this.parameters = buildSummary.parameters;
-    this.buildTools = buildSummary.build_tools;
-    this.cloneUrl = buildSummary.clone_url;
-    this.branch = buildSummary.branch;
-    this.ref = buildSummary.ref;
-    this.duration = buildSummary.duration;
-    this.sha = buildSummary.sha;
+  constructor(build: BuildResponse) {
+    this.number = build.number;
+    this.projectId = build.project_id;
+    this.description = build.description;
```

Comment:
I see `description` and `platform` are syntax highlighted, does this conflict with any system methods in JS?

Reply:
Hmm i just checked the reserved word list, don't see those on there. not sure why they're highlighted https://web.archive.org/web/20140902235313/http://javascript.about.com/library/blreserved.htm

Reply:
👍